BEPs/BEP322.md

```diff
@@ -288,6 +288,57 @@ Response:
 }
 ```
 
+##### 4.4.1.3 Query Bundle Data
+
+This API is used by mev data analysts to query the bundle information from the builder. For example, In order to be able 
+to directly reflect the mev market size, the mev data provider launches a bundle panel needs to show the number of 
+bundles per day or per block.
+
+Request:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "mev_bundles",
+  "params": [
+    {
+      "fromBlock": "0x1",
+      "toBlock": "0x2"
+    }
+  ],
+  "id": 1
+}
+```
+* The diff between fromBlock and toBlock should be less than 100.
+
+Response:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "result": [
+    {
+      "blockNumber": "0x1",
```

Comment:
does `blockNumber` means when receive this block?

Reply:
refact the response to present bundle's lifetime